.github/kokoro/kokoro-config.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
This repo's config is simple enough you don't really need a Python script to generate it.

Reply:
Removed.